conans/test/integration/basic_build_test.py

```diff
@@ -15,16 +15,18 @@ class BasicBuildTest(unittest.TestCase):
 
     def complete_build_flow_test(self):
         """In local user folder"""
-        files = cpp_hello_conan_files("Hello0", "0.1")
-        client = TestClient()
-        client.save(files)
-        command = "say_hello" if platform.system() == "Windows" else "./say_hello"
+        command = "bin\\say_hello" if platform.system() == "Windows" else "./bin/say_hello"
```

Comment:
This should work:
```python
command = os.sep.join([".", "bin", "say_hello"])
```
I suggest a test scope wide method, to be called as:
```python
command = os_command("bin", "say_hello")
```
In any case, please use the same in all occurences of this pattern